blogs/13_bfcl_v3_multi_turn.html

```diff
@@ -748,73 +751,59 @@ <h4>5. API Code Validation (🧑‍💻+ 💻 )</h4>
-            response-based evaluation in multi-turn function calling.</p>
-
-          <h4>Why State-based Evaluation is More Effective</h4>
-          <p>State-based evaluation provides a more accurate reflection of real-world performance. In multi-turn
-            interactions, the goal is often to update a system’s state—whether it's controlling a vehicle, managing
-            files, or processing data. Instead of only checking if each individual function output is correct, we
-            compare the attributes of the system’s state after every turn against the expected state. If the model
+          <h3 id="evaluation">Multi-turn Evaluation Metrics</h3>
+          <p>In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, we employed <strong>state-based</strong> evaluation and <strong>response-based</strong> evaluation to assess the model's performance the multi-turn categories. </p>
+          At the end of <strong>every</strong> turn, we mark an entry as correct if it passes both checks in all turns. Note that force-terminated entries will be marked wrong, even if they pass the checks.
+
+          <ol>
+            <li>
+              <p>
+                <strong>State-based</strong> evaluation focuses on comparing the backend system's state (excluding the private attributes, i.e., the ones that start with <code>_</code>) after all function calls are executed at the end of each turn of the conversation. We expect that given a user request, there can be multiple ways to fulfill the demand, which we are not able to measure, but the end state, or end result, should be consistent with ground truth labelling. The <strong>state-based</strong> evaluation capture the correctness of model executions that modify the internal state via <strong>write and delete</strong> e.g. create a new file, delete a stock from watchlist.</p>
+            </li>
+            <li>
+              <p><strong>Response-based</strong> evaluation compares the model's execution path against the minimial viable execution result paths as labeled in ground truth. The minimial viable execution result paths refer to a list of function calls that must be executed in order to produce desired response as user requests. Having <strong>response-based</strong> evaluation ensure <strong>read only</strong> request can be properly evaluated e.g. get the stock price, get the weather information.
+              </p>
+            </li>
+          </ol>
+
+          <p> In the
+            following sections, we will discuss the advantages and limitations of state-based evaluation in multi-turn function calling and why we need a subset-matched response-based evaluation as well.</p>
+
+          <h4>Why State-based Evaluation</h4>
+          <p>State-based evaluation provides a more accurate reflection of real-world performance as autonomous agents can detour on its own discreet while achieving the tasks after all. Instead of only checking if each individual function output is correct, we
```

Comment:
wording (more accurate reflection)

can rephrase this sentence. we are not claiming state-based is "better" or "more accurate". It's one dimension of measure